machAeroTutorials/overset_theory.rst

```diff
@@ -0,0 +1,158 @@
+.. _overset_theory:
+
+########################
+Overset Theory
+########################
+
+Overset Mesh
+=============
+
+ADFlow can only process multiblock structured meshes. For simple geometries, this is fine. 
+But it can be really hard to generate a single structured mesh for a complex geometry. 
+It might even be impossible to archive the required mesh quality.
+
```

Comment:
Hi Marco, thanks for taking a look. I agree with all of your points and will update them.

I will also change the first person stuff. I just wrote how it came to my mind withouth thinking about this.

It would be great if you somebody could work through it as a test. OpenVSP is completley free. I believe there is a trial version of pointwise that might be handy.